app.json

```diff
@@ -4,6 +4,9 @@
     "postdeploy": "rake db:migrate && rake db:seed"
   },
   "env": {
+    "GOOGLE_OAUTH2_CLIENT_ID": "131929309054-cb05cpsv068jhimnv0v6dqih3uagb0p5.apps.googleusercontent.com",
+    "GOOGLE_OAUTH2_CLIENT_SECRET": "Tb5DrpnwJzUiZ8y1GIkTgh8F"
+
```

Comment:
For Heroku, I think you just need this: 

```
"env": {
    "GOOGLE_OAUTH2_CLIENT_ID": { required: true }
    "GOOGLE_OAUTH2_CLIENT_SECRET": { required: true }  
}
```

We can define those ENV vars in the staging app settings and they will get pulled into the review app.